init.el

```diff
@@ -123,7 +123,8 @@ by Prelude.")
 (if (file-exists-p prelude-modules-file)
     (load prelude-modules-file)
   (message "Missing modules file %s" prelude-modules-file)
-  (message "You can get started by copying the bundled example file"))
+  (message "You can get started by copying the bundled example file")
+  (message "in sample/prelude-modules.el"))
```

Comment:
Because I preferred it to be in a new line, but that is a matter of personal taste.